tools/codegen/model.py

```diff
@@ -527,7 +527,6 @@ def kind(self) -> SchemaKind:
         else:
             return SchemaKind.functional
 
-    # WARNING: This method is not currently tested in any meaningful way
```

Comment:
It is properly tested now? :D

Reply:
I think @ezyang created this method for the autograd codegen work, but it was not integrated until this PR.
We expect it to generate the same canonical signature string for inplace/out/functional versions of the same function (so we can group them together). It seems to work so far :)